src/__tests__/rules.test.ts

```diff
@@ -48,19 +48,38 @@ describe('rules', () => {
   });
 
   it('should export configs that refer to actual rules', () => {
+    const expectJestPlugin = expect.objectContaining({
+      meta: {
+        name: 'eslint-plugin-jest',
+        version: '27.8.0',
```

Comment:
this should not be hard coded as it'll break for every new release

Reply:
oh yeah good point 🙈